k3po-maven-plugin/src/main/java/org/kaazing/k3po/maven/plugin/internal/StartMojo.java

```diff
@@ -52,7 +52,7 @@
     /**
      * @parameter name="control" default-value="tcp://localhost:11642"
      */
-    private URI controlURI;
+    private URI control;
```

Comment:
I had to move to maven-plugin-plugin to 3.4 (because of jdk8) and that seems to have broken this.